codexctl/updates.py

```
@@ -1,436 +1,459 @@
+        """Updates the version-ids.json file
+
+        Args:
+            location (str): Location to save the file
+
+        Raises:
+            SystemExit: If the file cannot be updated
+        """
+        with open(location, "w", newline="\n") as f:
+            try:
+                self.logger.debug("Downloading version-ids.json")
+                contents = requests.get(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)